### ISSUE TITLE
Missing 'viewport' problem was not shown near <meta> element

### ISSUE THREAD

**Issue by ngaitonde** [microsoft/vscode-edge-devtools#1003]
**Is your feature request related to a problem? Please describe.**

My html page had a <meta> element in <head>, but the problem for a missing 'viewport' wasn't shown inline. Instead, it was shown at the root <!DOCTYPE> element. 

**Describe the solution you'd like:**

I'd like to see problem next to the <meta> element, similar to the ''<html> element must have a lang attribute...' problem. See both attached images to compare the difference.

**Describe alternatives you've considered:**

**Additional context:**

![WHIssue2](https://user-images.githubusercontent.com/27184291/166166215-f4d0f910-ed3c-491a-a354-a27f026a79c9.png)
![WHIssue1](https://user-images.githubusercontent.com/27184291/166166288-4db659ce-22b5-4dff-9e2a-e33d37e8b244.png)

**Comment by codepo8** [comment]
Wouldn't the <head> element make more sense? Meta needs to be a child of that one.

**Comment by antross** [comment]
Yeah, I think pointing this at the `<head>` element (if present) would make more sense, the `<html>` element otherwise, and finally falling back to the start of the document if no `<head>` / `<html>` element is present.
